src/nft/components/details/detailsV2/TraitRow.tsx

```diff
@@ -0,0 +1,47 @@
+import Column from 'components/Column'
+import Row from 'components/Row'
+import { Trait } from 'nft/types'
+import { formatEth } from 'nft/utils'
+import styled from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+import { RarityGraph } from './RarityGraph'
+
+const SubheaderTiny = styled.div`
+  font-size: 10px;
```

Comment:
nit: Do we have any ThemedText that works here? If not, does the font size used in Figma have an actual name? Can we create a new ThemedText for it then? If yes, can we already pin down other occurrences of such configuration where we could use ThemedText too?

Reply:
This doesn't fit any current ThemedText styles. It's also the first occurrence I have personally seen of this typography and it is also not specced out in the global design system. If Design plans on making this a standard than we can add it but given this is currently a one-off I think current implementation is sufficient

Reply:
Yes, that makes sense - I agree. I have raised this with the design team tho.